### PR TITLE
Add attrs=false to README.md

### LINKED ISSUE #22: attrs=false should be documented
I handle lazy loading images on the client and this took a while to find

### PR DESCRIPTION
**Pull request by filipesilva**
Fix #22 .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +44,9 @@ require("html?attrs=img:src img:data-src!./file.html");
 require("html?attrs[]=img:src&attrs[]=img:data-src!./file.html");
 // => '<img  src="http://cdn.example.com/49e...ba9f/a9f...92ca.jpg"  data-src="data:image/png;base64,..." >'
 
+require("html?attrs=false!./file.html");
```
Comment (sokra) [inline README.md]: `html?-attrs` instead of `attrs=false` (this passes the string `false`)

Comment (filipesilva) [inline README.md]: Roger, it's in.

**Comment by sokra** [comment]
Thanks

**Comment by filipesilva** [comment]
Thank you for the great loader :D

**Pull request merged**